pkg/pool-stable-phantom/contracts/StablePhantomPool.sol

```diff
@@ -511,96 +521,80 @@ contract StablePhantomPool is StablePool, ProtocolFeeCache {
     }
 
     /**
-     * @dev Collects due protocol fees
-     */
-
-    function _collectProtocolFees()
-        private
-        returns (
-            uint256 bptOut,
-            uint256[] memory amountsIn,
-            uint256[] memory dueProtocolFeeAmounts
-        )
-    {
-        uint256 totalTokens = _getTotalTokens();
-
-        // This join neither grants BPT nor takes any tokens from the sender.
-        bptOut = 0;
-        amountsIn = new uint256[](totalTokens);
-
-        // Due protocol fees are all zero except for the BPT amount, which is then zeroed out.
-        dueProtocolFeeAmounts = new uint256[](totalTokens);
-        dueProtocolFeeAmounts[_bptIndex] = _dueProtocolFeeBptAmount;
-        _dueProtocolFeeBptAmount = 0;
-    }
-
-    /**
```

Comment:
Try to avoid moving blocks around in complicated PRs, as it makes the review unnecessarily harder (e.g. I had to check whether this block had changed at all).